src/certificate.rs

```diff
@@ -400,3 +401,49 @@ mod read_pki {
         }
     }
 }
+
+///Write information about certificate found in slot a la yubico-piv-tool output.
+pub fn print_cert_info(yubikey: &mut YubiKey, slot: SlotId) -> Result<(), Error> {
```

Comment:
This feels like it should be something like a `Debug` or `Display` impl, but offhand I'm not sure the right way to do that...

(or perhaps it should just exist in the CLI)

Reply:
Just in the CLI is fine. I already wrote it in the CLI (after the certificate read method added in parallel was pointed out to me). Will resubmit minus this, with the Sha256::digest suggestion but not CccCardId suggestion.